### ISSUE TITLE
[modal-cortex-s-100m-2b-preflight-v9-fused-linear-ce-compile-boundary]

### ISSUE THREAD

**Issue by vinceackermann2-sys** [vinceackermann2-sys/tam-research#895]
{"phase":"preflight-v9-fused-linear-ce-compile-boundary","source_sha":"e44dac49ccda2180d609c4189f08c9d7c5494392","source_tree":"7a7291912f87b89d2a4bb8f4cbe44e301f6c10be","harness_sha":"358208473b27b15c7f70e1985c1bdb1715b45767"}

**Comment by vinceackermann2-sys** [comment]
## Authoritative v9 H100 result — SYSTEMS PREFLIGHT FAIL

Sole paid attempt: workflow run `34623141185`, job `103341715882`, run #1 / attempt 1. No rerun or duplicate trigger is permitted.

Durable result classification: `ENGINEERING_SYSTEMS_PREFLIGHT_V9_FUSED_LINEAR_CE_COMPILE_BOUNDARY_ONLY`
Status: `SYSTEMS_PREFLIGHT_FAIL`

Frozen measurements:
- execution: `compiled`
- compile mode: `max-autotune-no-cudagraphs`
- compile seconds: `333.042615788`
- warmup steps: `5`
- measured optimizer steps: `40`
- measured tokens: `2,621,440`
- measured seconds: `12.60730991700001`
- training throughput: `207,930.16252144205 tok/s`
- projected pure training: `9,618.614133453377 s`
- frozen projected full envelope: `10,913.518162586715 s`
- required maximum: `8,500 s`
- peak VRAM: `28.145944118499756 GiB`
- last train loss: `7.240416526794434`

Gate results:
- finite loss: PASS
- exact parameter count: PASS (`101,778,112`)
- peak VRAM <=70 GiB: PASS
- positive throughput: PASS
- projected full seconds <=8500: **FAIL**

Backend proofs:
- Triton recurrent worlds: candidate calls `24/24`, legacy fallback `0`, all worlds selected Triton
- fused CUDA proof: Liger `0.8.2` selected on CUDA; CORTEX feature trunk compiled; Liger loss executes eager behind the explicit Dynamo boundary; full CUDA logits not materialized; fallback runner normalized; tied LM-head identity preserved

At this compile time, crossing the frozen 8,500-second gate would require approximately `269,378 tok/s`, about **29.55% higher throughput** than measured v9.

Interpretation: the v9 compile-boundary repair fixed the v8 incompatibility and produced a valid calibration, but the eager Liger boundary is materially slower end-to-end than the prior production path. It is therefore not a viable systems candidate under the frozen budget.

Engineering seed `2026091011`, this trigger title, and `/vol/cortex-s-v0/100m-2b/preflight-v9-fused-linear-ce-compile-boundary` are permanently consumed. Seed `8100` and scientific seeds `48131/48132/48133` were not used. No full 2B run or scientific claim is authorized.
